code/modules/food_and_drinks/food/snacks.dm

```diff
@@ -788,8 +788,8 @@
 	filling_color = "#B8E6B5"
 	list_reagents = list("nutriment" = 10, "vitamin" = 1)
 
-/obj/item/weapon/reagent_containers/food/snacks/soylentviridians
-	name = "Soylent Virdians"
+/obj/item/weapon/reagent_containers/food/snacks/soylenviridians
```

Comment:
I'm not going to even bother with each case, but you seriously messed up this entire file because you didn't properly update your code prior to this, which results in the system thinking you are attempting to undo the changes made to this file to fix some of the paths and descriptions.

This, as a result, leads to the TRAVIS builds failing because you've suddenly changed the paths of the spaghetti items on the map to no longer be valid objects.